lib/sessions.js

```diff
@@ -474,7 +474,7 @@ function endTransaction(session, commandName, callback) {
   }
 
   if (
-    commandName === 'commitTransaction' &&
+    // Assumption here that commandName is "commitTransaction" or "abortTransaction"
```

Comment:
Is there a way to verify this assumption?

Reply:
the function is currently only called with `'commitTransaction'` and `'abortTransaction'`. So as of right now, we don't need to check. I left the comment in case we ever add a 3rd option.